### PR TITLE
Make ping timestamp format configurable

### PR DESCRIPTION
**Pull request by prburgu**
### What is the purpose of this pull request?
The anycable-go service sends the ping messages on websocket connections. The message itself is a unix timestamp in seconds. We want to use this timestamp information to calculate the perceived latency of the messages from client's perspective. However the timestamps in seconds do not provide the precision we need. The timestamps in milliseconds is more desirable. To make the change to milli seconds backward compatible, allow ping timestamp format to be configurable.

### What changes did you make? (overview)
Added ping_timestamp_format config which takes the values of "ms" (milli seconds), "ns" (nano seconds) and "s" (seconds) with default "s". Based on the config specified, the ping messages will emit timestamps in milli seconds, nano seconds or seconds. 

### Checklist

- [ ] I've added tests for this change
- [ ] I've added a Changelog entry
- [ ] I've updated documentation

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -91,6 +91,8 @@ func init() {
 	fs.IntVar(&defaults.App.PingInterval, "ping_interval", 3, "")
 	fs.IntVar(&defaults.App.StatsRefreshInterval, "stats_refresh_interval", 5, "")
 
+	fs.StringVar(&defaults.App.PingTimestampFormat, "ping_timestamp_format", "s", "")
```
Comment (palkan) [inline cli/cli.go:94]: Let's call it `ping_timestamp_precision`.

**Review by palkan — COMMENTED** [review]
LGTM.

Left one comment (and a rebase required).

File: cli/cli.go (hunk 1 above)
Comment (prburgu) [reply cli/cli.go]: @palkan I've made this change.

**Review by palkan — APPROVED** [review]
Thanks!

Could you please add a change log entry?

**Review by palkan — APPROVED** [review]
Thanks!

**Pull request merged**